schema/codelists/extendedProcurementCategory.csv

```diff
@@ -1,5 +1,5 @@
 Code,Title,Description
-goods,Goods and supplies,This contracting process involves physical or electronic goods or supplies.
-works,Works,"This contracting process involves construction, repair, rehabilitation, demolition, restoration or maintenance of some asset or infrastructure."
-services,Services,"This contracting process involves professional services of some form, generally contracted for on the basis of measurable outputs or deliverables. When the consultingServices code is also available or in use for data from a particular dataset, the service code must only be used for non-consulting services."
-consultingServices,Consulting services,This contracting process involves professional services provided on a consultancy basis.
+goods,Goods and supplies,This contracting (or planning) process involves physical or electronic goods or supplies.
```

Comment:
For this codelist, the planning process doesn't involve goods/construction/etc. It's the future contracting process that will involve those.

What if we instead just change "This" to "The"? That way, from perspective of a planning process, it refers to the (future) contracting process. From the perspective of the contracting process, it refers to itself.

Going back to my comments in `documentType.csv`, maybe we could just state somewhere:

> When the description of a field or code refers to a contracting process:
>
> * If the release is for a contracting process, "contracting process" refers to itself.
> * If the release is for a planning process, "contracting process" refers to a future contracting process that will be related to the planning process.

Reply:
> For this codelist, the planning process doesn't involve goods/construction/etc. It's the future contracting process that will involve those.

Hm, for me already the planning process does? I guess I understand "involves" as "is about", so I read the codes as "process is about this stuff we wan't to buy". A contracting process then may end up with a contract with might end up with actual implementation, while a planning process may just end up with a contracting process, but that seems fine by me. 

____ 

> What if we instead just change "This" to "The"? That way, from perspective of a planning process, it refers to the (future) contracting process. From the perspective of the contracting process, it refers to itself.

I don't have a big issues with this, but I don't see the need that much either, see above. Also, even if it is "the contracting process", there is still the issue of "involves" vs. "will involve". 

(We are also inconsistent in the use of this/the for contracting/planning processes, the alignment of which could be another issue.) 

___ 

> When the description of a field or code refers to a contracting process:

If the release is for a contracting process, "contracting process" refers to itself.
If the release is for a planning process, "contracting process" refers to a future contracting process that will be related to the planning process.

Same as above - not hugely against, but don't have a strong urge to do it, either. We should still make sure it makes sense in individual fields, because people are unlikely to remember a "general" disclaimer as such.

Reply:
We can change to "is about" (or maybe better "is for"?), and keep the "(or planning)".

The primary meaning of "Involve" is "include (something) as a necessary part or result" – but no works, etc. result from a planning process.

Reply:
I've added "is for" in https://github.com/open-contracting/standard/pull/1513/commits/de94047a0a4980c4699a238553592a669eb0a424 and "the instead of this" in https://github.com/open-contracting/standard/pull/1513/commits/da953c4ab1e177015914dfda9f39c107fe09ec45. 

Also, I've opened https://github.com/open-contracting/standard/issues/1525.